src/open_inwoner/templates/pages/cases/status_inner.html

```diff
@@ -58,19 +58,29 @@ <h2 class="h2" id="documents-upload">{% trans "Document toevoegen" %}</h2>
                 {% endbutton_row %}
             {% endif %}
 
-            {#  Documents. #}
+            {#  Documents #}
             {% if case.documents %}
                 <h2 class="h2 {% if case.new_docs %}indicator{% endif %}" id="documents">{% trans 'Eerder toegevoegde documenten' %}</h2>
                 {% case_document_list case.documents %}
             {% endif %}
 
-            {#  Contact moment form #}
-            {% if case.contact_form_enabled %}
+            {# Questions/contactmomenten #}
+            {% if case.questions %}
+                {% include "pages/questions/questions.html" with questions=case.questions only %}
+                <div class="dropdown dropdown--spaced">
```

Comment:
This should be <div class="expand"> so you do not need to change the dropdown CSS, and move .hide-me to .card.